proto/proto-backplane-grpc/build.gradle

```diff
@@ -18,7 +18,7 @@ configurations {
 }
 
 // TODO(deephaven-core#1685): Create strategy around updating and maintaining protoc version
-def protobufVersion = '3.17.3'
+def protobufVersion = '3.20.1'
```

Comment:
We may need to update a package.json too?

Reply:
Searching our codebase for 3.17.3, I also see py/client and sphinx set 3.17.3

Reply:
updating the package.json requires running npm install in order to have it update the package-lock.json as well

Reply:
Updated, PTAL.

Reply:
Done.